swift/ql/lib/codeql/swift/generated/decl/ConcreteVarDecl.qll

```diff
@@ -7,6 +7,9 @@ module Generated {
   class ConcreteVarDecl extends Synth::TConcreteVarDecl, VarDecl {
     override string getAPrimaryQlClass() { result = "ConcreteVarDecl" }
 
+    /**
+     * Gets the introducer int.
```

Comment:
This one is not very self explanatory and could do with being expanded.

Reply:
definitely, it was my first candidate for an example, but actually this might be something we might want to take out of the extraction altogether. This is telling whether syntactically the variable was introduced with `var` or `let`. That affects settability, but there is other information that we should extract that tells us that, we should probably not really care whether syntaically the programmer has written `var` or `let`.

Reply:
I can imagine stylistic queries that care whether the programmer wrote `var` or `let`.

Reply:
added by b65f49bd50eed1828f81eab3d9bb2432c7bc4d83